Trumpf.Coparoo.Web/Controls/Table/Table.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
+{
+    using OpenQA.Selenium;
+
+    /// <summary>
+    /// Table control object.
+    /// </summary>
+    public class Table : ControlObject
+    {
+        /// <summary>
```

Comment:
What about adding one getter `Rows => Head.Row.Concat(Body.Rows).Concat(Foot.Rows)`? If someone does not care about whether there is a head or not etc.

Reply:
`AllRows => FindAll<Row>();` isn't this more intuitive or is there a downside to it?

Reply:
Only if the search does not guarantee that header rows appear before body rows etc. Can you check that? If that's the case you can use the `FindAll<Row>`.

Reply:
`AllRows => FindAll<Row>().OrderBy(r => r.Location.Y);` This should do it. Should we also add the ordering to the individual segments (head, body, foot)?

Reply:
Before you add that check wheter it is really necessary as this will significantly slow down performance: ordering requires to evaluate all rows. Hence calling `Rows.First()` is as slow as calling `Row.Last()`. By the way, I guess there is an extension `OrderFromLeftToRight` and `OrderFromTopToBottom`that does exaclty what you wrote.

Reply:
I just tested it and we actually need some sort of ordering since it's possible to declare the `tfoot` element before the `tbody` element within the HTML file (The footer will still appear at the bottom of the table)

Reply:
So we are back to `Head.Row.Concat(Body.Rows).Concat(Foot.Rows)`, right? This would suffice if each of the `Rows` is ordered from top to bottom by default, which I guess is the case. Can you confirm that (you have a lot of table tests as far as I remember)?

Reply:
Yes, I think `Head.Row.Concat(Body.Rows).Concat(Foot.Rows)` should be enough for now